nss-day-cohort-50/rare-client-rambunctious-rattlers
# Mc view tags

## Functionality
User can now click on `Tag Management` in NavBar and view a list of tags ordered alphabetically.

### *How to test*
Fetch both the client-side and server-side branches. Make sure your debugger is running. If there are no tags in your database, add some.
Click on `Tag Management` in the React app in the browser. You should see a list of tags.
